journeyapps-db/src/queryOperations.ts

```diff
@@ -117,6 +123,15 @@ export class Operation implements Expression {
     } else if (this.operator == 'contains') {
       if (typeof attributeValue == 'string' && typeof this.value == 'string') {
         return attributeValue.toLowerCase().indexOf(this.value.toLowerCase()) >= 0;
+      } else if (Array.isArray(attributeValue) && Array.isArray(this.value)) {
+        for (let child of this.value) {
```

Comment:
This is probably for my own interest sake,  but does this mean that the `contains` operator on multiple choice variables has to match each element specified in the query - ie, there is no built in way to check if the value contains any of the specified values in the query?

Reply:
Ah never mind, that is the `in` operator